src/tools/pub/package-layout.md

```diff
@@ -69,16 +69,15 @@ enchilada/
     Leave it out of source control unless your package is an
     [application package](/tools/pub/glossary#application-package).
 
-\*** The `packages` directories are
-     created when you run `pub get`,
-     unless you specify the flag `--no-packages-dir`.
+\*** The `packages` directories are created
```

Comment:
I think we can probably remove this bullet point too. I don't expect any users to pass `--packages-dir` who aren't already familiar enough with Dart development not to need this note.